modules/data-streams/src/yamlRestTest/java/org/elasticsearch/datastreams/DataStreamsClientYamlTestSuiteIT.java

```diff
@@ -50,6 +50,9 @@ private static ElasticsearchCluster createCluster() {
             .setting("xpack.security.enabled", "true")
             .keystore("bootstrap.password", "x-pack-test-password")
             .user("x_pack_rest_user", "x-pack-test-password");
+        if (initTestSeed().nextBoolean()) {
```

Comment:
I think we just need to use `nextBoolean()` here?

Reply:
It's not possible to use this function here since the executing thread doesn't contain a `RandomizedContext`. This is a workaround for using the random seed in early initialization logic.

Reply:
gotcha, yes that is tricky.